docs/secure-connections/sauce-connect/setup-configuration/environment-variables.md

```diff
@@ -8,21 +8,133 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+[`--no-autodetect`](/dev/cli/sauce-connect-proxy/#--no-autodetect) command-line option disables system proxy detection.
 :::
 
-| Environment Variable  | Description  | Platforms  | Corresponding CLI Option  |
-|---|---|---|---|
-| `SAUCE_USERNAME` | Sets your Sauce Labs username. | Windows, Linux, Mac | [`--user`](/dev/cli/sauce-connect-proxy/#--user) |
-| `SAUCE_ACCESS_KEY` | Sets your Sauce Labs access key. | Windows, Linux, Mac | [`--api-key`](/dev/cli/sauce-connect-proxy/#--api-key) |
-| `http_proxy`<br/>`HTTP_PROXY`<br/>`all_proxy`<br/>`ALL_PROXY` | Sets an HTTP proxy to be used by Sauce Connect Proxy. It can be formatted as `http://hostname:port` or `hostname:port`.<br/><br/>On Linux and Mac environments, `http_proxy` and `https_proxy` variables can contain proxy credentials in the following format: `scheme://user:password@host:port` | Windows, Linux, Mac | [`--proxy`](/dev/cli/sauce-connect-proxy/#external-proxy-configuration) |
-| `no_proxy`<br/>`NO_PROXY` | Sets hostnames that will not be proxied, even when Sauce Connect Proxy is configured to use a proxy. Format as a comma-separated list. Subdomain wildcarding is supported when the hostname starts with a `.` Examples:<br/><br/>`saucelabs.com,spam.net`: Only requests hitting `saucelabs.com` and `spam.net` will not be proxied. All other requests will be proxied.<br/><br/>`.example.com`: All requests going to any subdomain of `example.com` will not be proxied (e.g., `one.example.com`, `asdf.example.com`) | Linux | n/a |
-| `SAUCE_ARES_FLAGS` | Sets advanced flags for DNS library use with Sauce Connect Proxy. For more information, see the [c-ares Documentation](http://c-ares.haxx.se/ares_init.html). | Windows, Linux, Mac | n/a |
 
-For best performance, use the [latest version of Sauce Connect Proxy](/secure-connections/sauce-connect/installation).
+## What You'll Need
+* See [Sauce Connect Proxy Basic Setup requirements](/secure-connections/sauce-connect/setup-configuration/basic-setup/#what-youll-need).
+
+
+## Running Sauce Connect Proxy with enviroment variables
+
+  <Tabs
+      defaultValue="Mac/Linux"
+      values={[
+        {label: 'Mac/Linux', value: 'Mac/Linux'},
+        {label: 'Windows', value: 'Windows'},
+      ]}>
+
+<TabItem value="Mac/Linux">
+
+Follow the steps below to configure Sauce Connect Proxy using enviroment variables in the current terminal.
+
+1. In your terminal window, set the following enviroment variables
+  ```bash
+  export SAUCE_USERNAME="your Sauce username"
+  export SAUCE_ACCESS_KEY="your Sauce access key"
+  ```
+2. Starting a new Sauce Connect Proxy will not require adding `--api-key` or `--user` keys.
```

Comment:
This is a side comment, but I have noticed that we use the terms `flags` and `keys` somewhat interchangeably when we talk about the command line options. We should probably choose one and be consistent.

Reply:
I've been using the term _flags_ for CLI options (i.e., `--user`, like in this case) and _keys_ for YAML (i.e., `user:`). What do you think, @sweeneyskirt-sl ?

Reply:
I would support that usage.